### PR TITLE
Replace code with swift wheel

### PR DESCRIPTION
**Pull request by tastelikefeet**

### CODE REVIEW

File: docker/Dockerfile.ubuntu [hunk 1]
```diff
@@ -1,6 +1,14 @@
 ARG BASE_IMAGE=reg.docker.alibaba-inc.com/modelscope/modelscope:ubuntu20.04-cuda11.3.0-py37-torch1.11.0-tf1.15.5-base
 FROM $BASE_IMAGE
 
+FROM UBUNTU/REDHAT/
```
Comment (wenmengzhou) [inline docker/Dockerfile.ubuntu:4]: remove this

File: docker/Dockerfile.ubuntu [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+
+RUN pip install numpy -U
+
+COPY .xxx xxxx
+
+
```
Comment (wenmengzhou) [inline docker/Dockerfile.ubuntu:10]: remove

File: modelscope/trainers/trainer.py [hunk 3]
```diff
@@ -9,6 +9,7 @@
 
 import json
 import torch
+from swift import PeftConfig, Swift, SwiftConfig
```
Comment (wenmengzhou) [inline modelscope/trainers/trainer.py:12]: move to line 274, inside the if condition, and add swift import chck util in import_util.py,  make swift an optional dependency of modelscope

File: tests/trainers/test_finetune_vision_efficient_tuning_swift.py [hunk 4]
```diff
@@ -4,12 +4,10 @@
 import tempfile
 import unittest
 
+from swift import AdapterConfig, LoRAConfig, PromptConfig
```
Comment (wenmengzhou) [inline tests/trainers/test_finetune_vision_efficient_tuning_swift.py:7]: move test case to swift

File: docker/Dockerfile.ubuntu (hunk 2 above)
Comment (tastelikefeet) [reply docker/Dockerfile.ubuntu]: deleted

File: tests/trainers/test_finetune_vision_efficient_tuning_swift.py (hunk 4 above)
Comment (tastelikefeet) [reply tests/trainers/test_finetune_vision_efficient_tuning_swift.py]: moved

Comment (tastelikefeet) [reply tests/trainers/test_finetune_vision_efficient_tuning_swift.py]: The case uses efficient trainer, so basicly it's a test case belonged here
So instead, i made the test case optional to run.

**Review by slin000111 — APPROVED** [review]
examples passed the test.

**Review by wenmengzhou — APPROVED** [review]

**Pull request merged**